app/src/main/java/de/danoeh/antennapod/adapter/ChaptersListAdapter.java

```diff
@@ -109,7 +109,7 @@ public View getView(final int position, View convertView, ViewGroup parent) {
                     if (link.length != 0) {
```

Comment:
it's all within an `if (action == MotionEvent.ACTION_UP || action == MotionEvent.ACTION_DOWN)` block, that's why I did what I did, but since you think it added to clarity I'm reverting my change.